src/api/CMakeLists.txt

```diff
@@ -28,6 +28,17 @@ add_library(migraphx_c
 set_target_properties(migraphx_c PROPERTIES EXPORT_NAME c)
 rocm_set_soversion(migraphx_c 3.0)
 
+# Check miopen find mode api
+include(CheckLibraryExists)
+check_library_exists(MIGRAPHX "migraphx_program_run_async" "src/api/include/migraphx.h" HAS_STREAM_SYNC)
```

Comment:
This should be in onnxrt. There is no point in us checking our headers as we already know what in them.

Reply:
Ah I see, I can remove this and add to the EP side then.

Reply:
Done. Saved copy locally and will add to EP changes